### PR TITLE
data:anuj-kumary

### PR DESCRIPTION
**Pull request by anuj-kumary**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3139"></a>

### CODE REVIEW

**Review by accodes21 — APPROVED** [review]
Looks great

File: data/anuj-kumary.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        "icon": "FaTwitter"
+      },
+      {
+        "name": "LinkedIn: Let's connect",
+        "url": "https://www.linkedin.com/in/anujkumaryadav05/",
+        "icon": "FaLinkedin"
+      }
+    ]
+  }
+
```
Comment (Panquesito7) [inline data/anuj-kumary.json:30]: There's an extra unneeded line here.
```suggestion
```

**Review by Panquesito7 — APPROVED** [review]
Overall, it is looking good. Thanks! 🎉

File: data/anuj-kumary.json (hunk 1 above)
Comment (eddiejaoude) [reply data/anuj-kumary.json:30]: Prettier will fix this after the merge

**Pull request merged**